### ISSUE TITLE
Remove sfml_game::aboutScreenText

### ISSUE THREAD

**Issue by richelbilderbeek** [richelbilderbeek/djog_unos_2018#215]
**Is your feature request related to a problem? Please describe.**

Currently, `sfml_game` has member variable:

```c++
class sfml_game
{
  //...
  Text aboutScreenText;

  //...
};
```

As there is already a class `sfml_about_screen`, 
this member variable has become obsolete and must be removed.

**Describe the solution you'd like**

 * Delete it
 * Delete all lines where it is used

**Describe alternatives you've considered**

None.

**Additional context**

None.

**Comment by richelbilderbeek** [comment]
All that needs to be done:

 * merge develop into your branch
 * get it to work
 * if builds OK, then merge your branch into develop
 * if develop breaks, fix develop
 * close this Issue

Pro-tip: juse `fix #215` at the end of your commit message. The Issue will close if the commit reaches `master`.

**Comment by richelbilderbeek** [comment]
Done :+1:
